packages/tasit-action/src/contract/Action.test.js

```diff
@@ -0,0 +1,12 @@
+import Action from "./Action";
+import Subscription from "./Subscription";
+
+describe("subscribing to an event before sending tx", () => {
+
+    it("gives error if no subscribed events", () => {
```

Comment:
@zbash75 This file ended up with four spaces of indentation, and we normally use 2.

@marcelomorgado I think we already have this in our linting rules - so we'll need to find a strategy to make sure that CircleCI runs for feature branches on other people's forks.

Reply:
1) Yes, running `npm run lint` before commit will fix that, but add a `prettier` plugin to the editor is recommended setup because it runs lint on every file saving.

2) We are tracking this issue already (https://github.com/tasitlabs/TasitSDK/issues/158) it has no milestone yet, which one should belongs?

Reply:
1. Yep. @zbash75 let me know if the instructions on how to set this up in the [CONTRIBUTING guide](https://github.com/tasitlabs/TasitSDK/blob/develop/docs/CONTRIBUTING.md) aren't clear enough

2. @marcelomorgado I made that issue be with milestone `0.0.2`

Reply:
Hey @pcowgill what do you think to move `list` to a pre-commit hook using that package: https://www.npmjs.com/package/pre-commit ?

Reply:
I think we should make it a pre-commit hook, yes. But we might consider these other options too https://prettier.io/docs/en/precommit.html

Reply:
I've created:
https://github.com/tasitlabs/TasitSDK/issues/231
and
https://github.com/tasitlabs/tasit/issues/120

Reply:
@Olivierlu1 Do you need any help running the linter on the changes in this PR?